styleguide/source/_patterns/02-molecules/press-release-logos.json

```diff
@@ -0,0 +1,76 @@
+{
+  "images": [
+    {
+      "alt": "alt text",
```

Comment:
Since this is a public repository that theoretically someone could test with a screenreader we may want to put appropriate alt text on images.